README.md

```diff
@@ -2,4 +2,52 @@
 
 *⚠️ Note: This repository is a WIP, "watch" it to keep up with updates!*
 
-This repository hosts Bokeh equivalents for various plots from ["Fundamentals of Data Visualization" by Claus O. Wilke](https://clauswilke.com/dataviz/).
+## Project description
+
+This repository hosts Bokeh equivalents for various plots from ["Fundamentals of Data Visualization" by Claus O. Wilke](https://clauswilke.com/dataviz/). It provides a collection of interactive data visualizations implemented using the Bokeh library.
+
+## Table of Contents (WIP)
+
+- Index - Contains a link to all of the available plots.
+- Notebook 1: [Plotting Bar Charts](notebooks/bar_charts.ipynb) - Demonstrates how to recreate the bar charts using Bokeh.
+- Notebook 2: [Scatter Plots](notebooks/scatter_plots.ipynb) - Shows how to generate the scatter plots with Bokeh.
+- Notebook 3: [Line Plots](notebooks/line_plots.ipynb) - Illustrates how to recreate the line plots using Bokeh.
+
+## Local Setup
+
+To run these notebooks locally, follow these steps:
+
+1. Clone the repository:
+
+    git clone https://github.com/bokeh/dataviz-fundamentals.git
+
+2. Navigate to the project directory via the terminal or command prompt.
+
+3. Create a new conda environment and install the required dependencies:
+
+    conda env create -n <name> -f environment.yml
+
+replacing `<name>` with your preferred environment name.
+
+4. Activate the new environment:
+
+    conda activate <name>
+
+5. Open Jupyter notebook via anaconda navigator or via the command line:
+
+    jupyter notebook
+
+6. Open the desired notebook in your web browser and run the cells.
+
+## Contributing
+
+Contributions are welcome! If you would like to contribute to this project, please follow the guidelines below:
+
+- Fork the repository and create your branch.
+- Make your changes and ensure the code follows the project's coding style.
+- Test your changes thoroughly.
+- Submit a pull request with a clear description of your changes.
+
+## License & Code of Conduct
+
+This project is licensed under the [BSD 3-Clause License](https://github.com/bokeh/bokeh/blob/branch-3.1/LICENSE.txt). By contributing to this project, you agree to abide by the [Bokeh Code of Conduct](https://github.com/bokeh/bokeh/blob/branch-3.1/docs/CODE_OF_CONDUCT.md).
```

Comment:
@bryevdv I see that the data and plots used in the book are under the [MIT license here](https://github.com/clauswilke/dviz.supp/blob/master/LICENSE), so I think we should be good to license this project under our regular BSD 3-Clause?

Reply:
Isaiah - To clarify, this is a question for @bryevdv

Reply:
AFAIK that is the case but IANAL :) We could also just release this repo under MIT I think that would also be fine

Reply:
Thanks! Looking at https://fossa.com/blog/open-source-licenses-101-mit-license/, I think we can:
- Keep the phrasing on line 53 here as is (i.e., continue using our Bokeh license)
- And, in addition, add a copy of https://github.com/clauswilke/dviz.supp/blob/master/LICENSE in this repo